smallslive/templates/home_banners.html

```diff
@@ -14,7 +14,7 @@
             <button id="banner-button"></button>
           </div>
           <div class="banner-learnmore-info hidden">
-            <button id="banner-button-info">More Info</button>
+            <button id="banner-button-info" onclick="moreInfo()">More Info</button>
```

Comment:
This should be `<a href="{% url 'contact-and-info' %}">More Info</a>`

Reply:
I would advise not to, there is a lot of mixins and css made for buttons inside the banner otherwise we would en duplicating a lot of css.